include/aws/sdkutils/resource_name.h

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+#ifndef AWS_COMMON_RESOURCE_NAME_H
+#define AWS_COMMON_RESOURCE_NAME_H
+#pragma once
+
+#include <aws/sdkutils/exports.h>
```

Comment:
trivial: I thought our convention was to always include <aws/libname/libname.h> from every other header in that lib? ... in which case why does exports.h exist ... and I don't know